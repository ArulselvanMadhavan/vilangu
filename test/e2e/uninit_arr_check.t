int main() {
  int a[];
  out a[0];  
}