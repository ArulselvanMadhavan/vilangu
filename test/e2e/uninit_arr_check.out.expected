Line: 3 | Null dereference
